OpenRA.Mods.Common/Activities/DeliverUnit.cs

```diff
@@ -214,5 +214,16 @@ public override bool Cancel(Actor self, bool keepQueue = false)
 
 			return base.Cancel(self);
 		}
+
+		public override TargetLineNode? TargetLineNode(Actor self)
+		{
+			if (innerActivity != null && innerActivity is HeliFly)
+			{
+				var a = innerActivity as HeliFly;
+				return new TargetLineNode(a.Target, Color.Yellow, true);
```

Comment:
Why do you need to create this explicitly?  This should be able to chain to the inner activity directly.

Reply:
I wasn't confident with chaining as I only saw this activity used in conjunction with HeliFly activity. I'll have it chained.